odtp/cli/new.py

```diff
@@ -133,13 +153,7 @@ def execution_entry(
         )
     except Exception as e:
         log.error(f"ERROR: {e}")
-        if hasattr(e, "__notes__"):
-            log.error(f"{','.join(e.__notes__)}") 
-            raise typer.Abort()     
-    log.info(f"""SUCCESS: execution has been added: see above for the details.
+        traceback.print_exc()
+    print(f"""SUCCESS: execution has been added: see above for the details.
```

Comment:
Shouldn't this use typer.echo as above? I didn't use `typer.echo` it looks like an interesting. As far as I understood it allows to use emojis and complex styling right?